salesforce/10.10/modules/ROOT/pages/salesforce-connector-studio.adoc

```diff
@@ -72,367 +74,242 @@ Subscribe to a topic. This source provides topic-related notification settings f
-+
-Number of the port on which the proxy server runs
-+
-** *Username*
-+
-Username to log in to the server
+* <<basic,Basic authentication>>
+* <<oauth,OAuth 2.0 authentication>>
+* <<oauth-jwt,OAuth JWT authentication>>
+* <<oauth-saml,OAuth SAML authentication>>
+* <<oauth-username-password,OAuth Username Password authentication>>
+
 +
-** *Password*
+*Kim. Ideally the next two paragraphs would be in a section called Authentication, but that would make the current H3's for the authentication types into H4s. I would still like to do that unless you have a better idea. I also am having trouble getting these paragraphs to format correctly within the list.* 
+
 +
-Corresponding password
-. On the *Advanced* tab, optionally specify reconnection information, including a reconnection strategy.
+For all OAuth connections you can specify an object store that stores each resource owner's ID data. If you don't specify an object store, Mule automatically provisions the default object store. The app interacts with the object store automatically when a new resource owner is authenticated, the access token is refreshed, or the access token is invalidated.
+
+You can <<mutual-tls,configure mutual TLS>> for all authentication methods.
+
+. In the *Apex* tab, optionally configure <<apexsets,Apex Settings>>.
+. In the *Advanced* tab, optionally specify reconnection information, including a reconnection strategy.
 . Click *Test Connection* to confirm that Mule can connect with the specified server.
 . Click *OK*.
 
-After completing the fields in the *Global Element Properties* screen, return to the main connector menu and set the <<Connector Property Values>>.
-
+[[basic]]
 === Basic Authentication
 
-Basic authentication uses a username and password for authentication.
+Basic authentication uses a username and password for authentication. Important fields include the following:
 
-Important fields include the following:
+[%header,cols="30s,70a"]
+|===
+|Field a|Description
+| Username| Salesforce username
+| Password | Corresponding password
+| Security token | Corresponding security token
+| Callback path| Path for the callback, for example, `/callback`
+| Authorization URL | URL for the endpoint configured to process SOAP authentication requests
+|===
 
-* *Username* (required) +
-Salesforce username
-* *Password* (required) +
-Corresponding password
-* *Security token* +
-Corresponding security token
-* *Authorization URL* +
-URL for the endpoint configured to process SOAP authentication requests
+[[oauth]]
+=== OAuth 2.0 Authentication
 
-=== OAuth 2.0
+OAuth 2.0 delegates user authentication to the service hosting the user account. Important fields include the following:
 
-OAuth 2.0 delegates user authentication to the service hosting the user account.
+[%header,cols="30s,70a"]
+|===
+|Field a|Description
+|Consumer key | Consumer key for the Salesforce-connected app. See the Salesforce documentation for information about generating a consumer key.
```

Comment:
Because this Description entry extends beyond a fragment, ALL entries in the column must end with periods.
For parallelism, this means that you should probably end all Description entries in all four peer sections with periods.
Unless you can get rid of the xref...?